OS/DiskOS/Programs/FileSystem/copy.lua

```diff
@@ -4,10 +4,10 @@
 local args = {...} --Get the arguments passed to this program
 if #args < 2 or args[1] == "-?" then
   printUsage(
-    "copy <source> <destination>","Copies a file creating any missing directory in the destination path"
+    "copy <source> <destination> [--verbose]","Copies a file or directory, creating any missing directory in the destination path"
```

Comment:
Okay, One thing I would recommend here is adding `-v`, Like in linux commands, they have short and long version of options.